motr/client_init.c

```diff
@@ -1527,8 +1527,13 @@ M0_INTERNAL int m0_client_global_init(void)
 	return M0_RC(rc);
 }
 
+/**
+ * To get the enviroment value for M0_CLIENT_ADDB_DIR
+ */
+#ifndef __KERNEL__
+#include "lib/string.h"
+#endif
```

Comment:
lib/string.h already has #ifndef __KERNEL__ in it. No need to check again here.
Also, please move all #include statements to the beginning of the file along with the others.